frontend/app/assets/stylesheets/spree/frontend/views/spree/products/index.scss

```diff
@@ -295,14 +295,19 @@
       }
     }
     &-buttons {
-      bottom: 30px;
+      background-color: $primary-background;
+      padding-bottom: 30px;
+      bottom: 0px;
     }
   }
 
   &-scroll {
     max-height: 100%;
     overflow-y: auto;
-    padding-bottom: 140px;
+    padding-bottom: 150px;
```

Comment:
Why 150px? The padding should be aligned to the buttons container.

On staging:
<img width="1014" alt="Zrzut ekranu 2020-04-3 o 11 55 07" src="https://user-images.githubusercontent.com/26117538/78348519-74072b00-75a2-11ea-971e-4b17803a8d62.png">

After changes:
<img width="1048" alt="Zrzut ekranu 2020-04-3 o 11 57 36" src="https://user-images.githubusercontent.com/26117538/78348528-779ab200-75a2-11ea-801a-f175818f1654.png">

Reply:
On mobiles, the buttons section has a height of `148px` so padding smaller than that will result in filters content being overlapped.